agr_literature_service/lit_processing/data_ingest/dqm_ingest/sort_dqm_json_reference_updates.py

```diff
@@ -277,6 +277,16 @@ def sort_dqm_references(input_path, input_mod, base_dir=base_path):      # noqa:
     missing_papers_in_mod = {}
     missing_agr_in_mod = {}
 
+    search_path = path.dirname(path.abspath(__file__)).replace("/dqm_ingest", "") + \
```

Comment:
This would be more portable:

search_path = join(dirname(dirname(abspath(__file__))), "pubmed_ingest", "data_for_pubmed_processing")
exclude_pmid_file = join(search_path, "pmids_to_excude.txt")